app/components/PageGrid.tsx

```diff
@@ -58,6 +60,19 @@ export function PageGrid({ left, subtitle, title }: PageGridProps) {
 						rotate={-45}
 					/>
 				</a>
+				<a
+					className="page-grid-footer-separator"
+					href="https://jawns.club/@jsclub"
+					rel="me noreferrer"
+					target="_blank"
+				>
+					Mastodon
+					<Arrow
+						className="arrow-out"
+						label="External link indication arrow"
+						rotate={-45}
+					/>
+				</a>
```

Comment:
[Design] The screenshots show this being left-aligned, which IMO doesn't look great. Can we figure out a visual design where it cleanly fits in?

Reply:
I missed that part of the span separator and then the next anchor tag; sorry 🤓 
Here's an updated view:
<img width="632" alt="Screen Shot 2023-05-01 at 14 42 30" src="https://user-images.githubusercontent.com/45559220/235509209-76913d8c-1c91-4186-8342-391926b23647.png">
<img width="176" alt="Screen Shot 2023-05-01 at 14 42 48" src="https://user-images.githubusercontent.com/45559220/235509213-48670ecc-8fed-4ba4-9b27-a7d9e3da1970.png">

Reply:
Looks pretty straightforward 👍🏼 -- it might make sense long term to move to something more like this in CSS Grid so these links don't break oddly on desktop, though. https://codepen.io/seriouslysean/pen/MWPEJvR?editors=1100
![image](https://user-images.githubusercontent.com/414475/235530882-31fd948e-317d-40f5-828a-f5a53d6f3ebe.png)

Reply:
#7 if anybody wants 😄 

Idea: what about having two lines of links? First line is relative (non-↗️) links, and the second line is the external (↗️) links? Having one link wrapping to the next line is triggering my orderly instincts a bit.

Reply:
There's not enough room for two lines on <= laptop screens; Maybe a desktop size will work.
I might try to reduce the font size, and it might give more possibilities, will check that one tomorrow.

Reply:
Ooh maybe 3 rows of 2 becoming 2 rows of 3... the possibilities! 😄  good luck!

Reply:
This is the most balanced option without dealing with CSS changes; I had to reorganize the links.
Let me know if it works for you

<img width="622" alt="image" src="https://user-images.githubusercontent.com/45559220/235720100-0a05174a-a646-44b7-abf0-b25571fdde4e.png">

<img width="186" alt="image" src="https://user-images.githubusercontent.com/45559220/235720262-d74317a9-fbd9-4965-be23-b7cdc00368cb.png">